docs/guides/admin/docs/configuration/security.cas.md

```diff
@@ -66,9 +66,9 @@ You will need to configure a [UserProvider](security.md) to look up users as ide
 * [Brightspace D2L User Provider](security.user.brightspace.md)
 * [Canvas LMS User Provider](security.user.canvas.md)
 
-
-Original documentation from University of Saskatchewan
-------------------------------------------------------
-
+Further Information
```

Comment:
We should fix this. I'll include the relevant docs in a follow-up pr.

Reply:
@gregorydlogan UPV is using CAS. Maybe we can ask Carlos to overhaul this guide… well… or make someone do it :grin:

Reply:
I've filed it and will get those docs tomorrow.